crates/consensus/primary/src/consensus_bus.rs

```diff
@@ -282,6 +282,11 @@ impl ConsensusBus {
         &self.inner.tx_consensus_round_updates
     }
 
+    /// Load the atomic GC round for consensus.
+    pub fn atomic_gc_round(&self) -> u32 {
```

Comment:
Given the convoluted path to get to this, this seems fine.  In general I like to keep helpers down on the bus so it does not become a confused pile of "helpers" by returning more generic things with the helpers (ie the TnSender/Reciever traits).  This is just a general comment though, in this case this may be the best option.